mirage/serializers/application.ts

```diff
@@ -29,4 +29,27 @@ export default class ApplicationSerializer extends JSONAPISerializer {
         const count = Array.isArray(related) ? related.length : 0;
         return relatedCounts.includes(this.keyForRelationship(relationship)) ? { count } : {};
     }
+
+    serialize(_: ModelInstance, __: Request) {
+        // eslint-disable-next-line prefer-rest-params
+        const json = JSONAPISerializer.prototype.serialize.apply(this, arguments);
```

Comment:
So I didn't go with `(model, request)` initially because I don't think I'm guaranteed that it's just the model and request that get passed. With the stuff that we do, yes, but I'm not sure if Mirage might add extra arguments just because. That being said, it's prooooobably fine.